modules/testlogger/testlogger.go

```diff
@@ -13,7 +13,6 @@ import (
 	"testing"
```

Comment:
Moving to `modules/log/testlogger`?

Reply:
No, that's unrelated change. Not related to the "rewriting logger system" itself.